.changes/incorrect-label-linux-command.md

```diff
@@ -0,0 +1,7 @@
+---
+"tauri": "minor:bug"
```

Comment:
I think we should release it as a patch instead

Reply:
tauri-runtime and tauri-runtime-wry probably can keep the minor as it's a breaking change

Reply:
yeah tauri could probably be `patch`